core/src/main/java/org/elasticsearch/index/SearchSlowLog.java

```diff
@@ -73,20 +73,17 @@
     public static final Setting<TimeValue> INDEX_SEARCH_SLOWLOG_THRESHOLD_FETCH_TRACE_SETTING =
         Setting.timeSetting(INDEX_SEARCH_SLOWLOG_PREFIX + ".threshold.fetch.trace", TimeValue.timeValueNanos(-1),
             TimeValue.timeValueMillis(-1), Property.Dynamic, Property.IndexScope);
-    public static final Setting<Boolean> INDEX_SEARCH_SLOWLOG_REFORMAT =
-        Setting.boolSetting(INDEX_SEARCH_SLOWLOG_PREFIX + ".reformat", true, Property.Dynamic, Property.IndexScope);
```

Comment:
The setting had an effect before 5.x. In 2.x and below we had access to the original text of the query on the shard level and therefore we could choose to write it into the slow log in the original format (possibly with new lines) or reformat it into a single line. In 5.x the query comes to us in a parsed way, so the original query is not longer available hence it makes little sense to keep this setting. In other words, we broke it in 5.0.0, now we return back original default behavior while removing the setting...